src/pages/Groups/Group1/index.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react'
 
+import GroupContent from '../../../components/GroupContent'
+import BookingOld from '../../../assets/Group1/booking_old.svg'
+import ScheduleOld from '../../../assets/Group1/scheduling_old.svg'
+
 const Group1 = () => {
-  return <>Group 1 page</>
+  return (
+    <>
+      <GroupContent
+        groupNumber={1}
+        names="Chun Yu, En Hao, Prince, Maxim, Richard, Shermaine, Nguyen, Felicia"
+        oldScreen1ImgSrc={BookingOld}
+        oldScreen2ImgSrc={ScheduleOld}
+      />
+    </>
```

Comment:
empty tags not required if there's only one component

Reply:
fixed in commit "b633ef9099261dc56d797b8bc08d096769891e48"